documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -1,45 +1,86 @@
 == Raspberry Pi Touch Display
 
-The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display which connects to the Raspberry Pi through the DSI connector. In recent versions of Raspberry Pi OS, both the HDMI and LCD display outputs can be used at the same time.
+The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display which connects to the Raspberry Pi through the DSI connector. If your Raspberry Pi has both connectors, you can use the micro HDMI and LCD display outputs at the same time.
+
+Make sure your Raspberry Pi OS is up-to-date before you begin working with the Touch Display to reduce the risk of errors or driver issues.
 
 === Board Support
 
-The DSI display is designed to work with all models of Raspberry Pi, however early models that do not have mounting holes (the Raspberry Pi 1 Model A and B) will require additional mounting hardware to fit the HAT-dimensioned stand-offs on the display PCB.
+The DSI display is designed to work with all models of Raspberry Pi, however early models without mounting holes (the Raspberry Pi 1 Model A and B) will require additional mounting hardware to fit the HAT-sized stand-offs on the display PCB.
```

Comment:
If we're tweaking this page anyway, might be worth updating "DSI Display" to say "Touch Display", for consistency?

Reply:
yes

Reply:
Done